scio-extra/src/main/scala/com/spotify/scio/extra/csv/CsvIO.scala

```diff
@@ -88,27 +89,47 @@ import org.apache.beam.sdk.values.PCollection
  */
 object CsvIO {
 
-  final val DefaultCsvConfig: CsvConfiguration = CsvConfiguration(
+  final private val _DefaultCsvConfig: CsvConfiguration = CsvConfiguration(
```

Comment:
I think the `_` prefix convention is for python.
I there are no name conflict it could have stayed without

Reply:
Was colliding with the WriteParam versions of `DefaultCsvConfig`, but using the full `CsvIO.DefaultCsvConfig` is better 👍